neofetch

```diff
@@ -1216,7 +1216,7 @@ get_song() {
     }
 
     case "${player/*\/}" in
-        "mpd"*)         song="$(mpc current)" ;;
+        "mpd"* | "mopidy"*)         song="$(mpc current)" ;;
```

Comment:
Just remove the spacing between `)` and `song=` and this will be perfect.

Example:

```sh
"mpd"* | "mopidy"*) song="$(mpc current)" ;;
```